wiki/Submission/en.md

```diff
@@ -34,91 +39,88 @@ You can also select the following via checkboxes under the message input box:
 
-### ![](heartpop.gif "fig:heartpop.gif") Disqualifying
+#### ![Heart-broken icon](/wiki/shared/Heartpop.gif "Heart-broken icon") Disqualifying
 
 Beatmap Nominators are only human. If a beatmap happens to be qualified and the QAT finds something that in their opinion prevents the beatmap from being officially approved, the beatmap can be temporarily disqualified and any scores achieved on it *wiped*. The beatmap is returned to WIP status. Once the issues are fixed, the beatmap can be qualified again.
 
-### ![](radioactive.gif "fig:radioactive.gif") Nuke
+#### ![Nuke icon](/wiki/shared/Radioactive.gif "Nuke icon") Nuke
 
 This symbol is given to beatmaps which **fail to follow even the most basic of guidelines**, such as having wildly incorrect timing or objects placed at random on the grid and/or timeline, and thus need a lot of improvement to even begin considering to Rank them.
 
-A Nuke symbol on a beatmap does not mean the moderators disapprove of it in general. It just means they disapprove of *its current condition*. If your beatmap is marked with a Nuke, don't worry - you can usually turn this around if you're determined:
+A Nuke symbol on a beatmap does not mean the moderators disapprove of it in general. It just means they disapprove of *its current condition*. If your beatmap is marked with a Nuke, do not worry - you can usually turn this around if you're determined:
 
 -   Pay close attention to the moderator's comments and suggestions in the thread.
--   Read the [Beatmapping](Beatmapping "wikilink"), Help pages, and/or watch tutorials on beatmapping.
+-   Read the [Beatmapping](/wiki/Beatmapping), Help pages, and/or watch tutorials on beatmapping.
 -   Open and view Ranked beatmaps in the Editor to see what they look and sound like.
 -   If you are new to beatmapping, it may be that the song you have chosen is challenging to time and/or map; in this case, you can also consider making a beatmap to an easier song. Try choosing songs with a constant, clear and steady rhythm to give yourself an easier time.
 -   Don't rush - as you keep trying, you'll constantly improve.
 
-### The Beatmap Graveyard
+#### The Beatmap Graveyard
 
 Threads on the Completed Beatmaps and Beatmap Help forums that do not receive replies within 4 weeks since their last post are automatically moved to the Beatmap Graveyard subforum to prevent the other beatmap subforums from getting cluttered. To prevent this from happening, you can use the Beatmap Submission System to reupload your beatmap and bump its thread in the process. Please use this feature once every few days and only if you have followed all moderator suggestions so far.
 
 Even if your beatmap gets sent to the graveyard, you can easily recover it from there if you wish to work on it again. Press the 'Resurrect' button on your profile.
 
-Criteria for ranking
---------------------
+### Criteria for ranking
```

Comment:
ditto

> imo, to prevent saying things multiple times, inconsistently, I'd merge the content of this section in with the Main article, then just say to go to that page ¯\_(ツ)_/¯